mobile/src/main/java/org/openhab/habdroid/core/ForegroundService.java

```diff
@@ -0,0 +1,80 @@
+package org.openhab.habdroid.core;
+
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.SharedPreferences;
+import android.os.IBinder;
+import android.preference.PreferenceManager;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.content.res.ResourcesCompat;
+import android.util.Log;
+
+import org.openhab.habdroid.R;
+import org.openhab.habdroid.ui.OpenHABMainActivity;
+import org.openhab.habdroid.util.Constants;
+
+public class ForegroundService extends Service {
```

Comment:
Wouldn't this be better named `CustomBroadcastListenerService`?

Reply:
I renamed it.